From 441512f4cf84e6d85046168b4fed7ec94e7e9050 Mon Sep 17 00:00:00 2001
From: mantid-builder <mantid-buildserver@mantidproject.org>
Date: Tue, 7 May 2019 15:03:08 +0000
Subject: [PATCH] clang-format PR25665 25f0523

---
 Framework/Algorithms/inc/MantidAlgorithms/CylinderAbsorption.h | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Framework/Algorithms/inc/MantidAlgorithms/CylinderAbsorption.h b/Framework/Algorithms/inc/MantidAlgorithms/CylinderAbsorption.h
index c19d437..9605ed4 100644
--- a/Framework/Algorithms/inc/MantidAlgorithms/CylinderAbsorption.h
+++ b/Framework/Algorithms/inc/MantidAlgorithms/CylinderAbsorption.h
@@ -97,11 +97,11 @@ private:
   void getShapeFromSample(const Geometry::IObject &sampleShape,
                           bool updateHeight, bool updateRadius);
   std::map<std::string, std::string> validateInputs() override;
-  std::vector<double> m_cylAxis;  ///< The axis orientation of the cylinder
-  double m_cylHeight;             ///< The height of the cylindrical sample in m
-  double m_cylRadius;             ///< The radius of the cylindrical sample in m
-  int m_numSlices;                ///< The number of slices
-  int m_numAnnuli;                ///< The number of annuli
+  std::vector<double> m_cylAxis; ///< The axis orientation of the cylinder
+  double m_cylHeight;            ///< The height of the cylindrical sample in m
+  double m_cylRadius;            ///< The radius of the cylindrical sample in m
+  int m_numSlices;               ///< The number of slices
+  int m_numAnnuli;               ///< The number of annuli
   bool m_useSampleShape;
 };
 
-- 
2.7.4

